src/Nordea/Components/Table.elm

```diff
@@ -0,0 +1,111 @@
+module Nordea.Components.Table exposing
+    ( tbody
+    , td
+    , th
+    , thead
+    , tr
+    , view
+    )
+
+import Css
+    exposing
+        ( alignSelf
+        , backgroundColor
+        , borderBottom3
+        , center
+        , column
+        , displayFlex
+        , flexDirection
+        , height
+        , hover
+        , nthChild
+        , padding
+        , padding4
+        , px
+        , rem
+        , row
+        , solid
+        )
+import Css.Global exposing (descendants, typeSelector)
+import Html.Styled as Html exposing (Attribute, Html)
+import Html.Styled.Attributes exposing (css)
+import Nordea.Components.Text as Text
+import Nordea.Resources.Colors as Colors
+
+
+view : List (Attribute msg) -> List (Html msg) -> Html msg
+view attrs children =
+    Html.table
+        (css [ displayFlex, flexDirection column ]
+            :: attrs
+        )
+        children
+
+
+{-| Groups the header content in a table
+-}
+thead : List (Attribute msg) -> List (Html msg) -> Html msg
+thead attrs children =
+    Html.thead
+        (css
+            [ height (rem 3.0)
+            , borderBottom3 (px 1) solid Colors.grayLight
+            ]
+            :: attrs
+        )
+        children
+
+
+{-| Defines a row in a table
+-}
+tr : List (Attribute msg) -> List (Html msg) -> Html msg
+tr attrs children =
+    Html.tr
+        (css
+            [ displayFlex
+            , flexDirection row
+            ]
+            :: attrs
+        )
+        children
+
+
+{-| Defines a header cell in a table
+-}
+th : List (Attribute msg) -> List (Html msg) -> Html msg
+th attrs children =
+    Html.th
+        (css [ padding4 (rem 1.0) (rem 0.75) (rem 0.75) (rem 0.75) ] :: attrs)
+        [ Text.init Text.TextSmallHeavy
+            |> Text.view [] children
+        ]
+
+
+{-| Groups the body content in a table
+-}
+tbody : List (Attribute msg) -> List (Html msg) -> Html msg
+tbody attrs children =
+    Html.tbody
+        (css
+            [ displayFlex
+            , flexDirection column
+            , descendants
+                [ typeSelector "tr"
+                    [ height (rem 4.5)
```

Comment:
Do we need to define a height? I'm thinking maybe padding is enough to get a default row height, and then the height property can be specified if needed when the table is used?
```suggestion
                    [ height (rem 4.5)
```

Reply:
Prooobably not. Just seemed like that height was default on all the examples. If the row height isn't set here, then we can't set default padding either, since it will affect text and icons inside the rows differently 🤔 
I can ask Thea if the table rows should have a set height or not 😊

Reply:
Prooobably not. Just seemed like that height was default on all the examples. If the row height isn't set here, then we can't set default padding either, since it will affect text and icons inside the rows differently 🤔
I can ask Thea if the table rows should have a set height or not 😊

Reply:
Hmm, will padding work if we add `alignItems center`? 🤔

Reply:
No, looks like it will still shrink to fit whatever content there is + padding 😬